ghost/content.md

```diff
@@ -1,33 +1,58 @@
 # Ghost
 
-Ghost is a free and open source blogging platform written in JavaScript and distributed under the MIT License, designed to simplify the process of online publishing for individual bloggers as well as online publications.
+Ghost is a free and open source blogging platform written in JavaScript and distributed under the MIT Licence, designed to simplify the process of online publishing for individual bloggers as well as online publications.
 
 > [wikipedia.org/wiki/Ghost_(blogging_platform)](http://en.wikipedia.org/wiki/Ghost_%28blogging_platform%29)
 
 %%LOGO%%
 
 # How to use this image
 
+This will start a Ghost instance listening on the default Ghost port of 2368.
+
 ```console
-$ docker run --name some-ghost -d ghost
+$ docker run —d --name ghostdemo ghost
 ```
 
-This will start a Ghost instance listening on the default Ghost port of 2368.
+### Custom port
 
 If you'd like to be able to access the instance from the host without the container's IP, standard port mappings can be used:
 
 ```console
-$ docker run --name some-ghost -p 8080:2368 -d ghost
+$ docker run —d --name ghostdemo -p 3001:2368 ghost
```

Comment:
1. The port, too many apps uses 8080 so I decided to use 3001. Purely random and not use in any containers I use.

2. I don't like `some-ghost` because I can't double click on it. I also prefer the see the word ghost first

3. `-d` I prefer to see it prior the name. Unwritten rule for me.